src/FakerDotNet/Data/CreditCardData.cs

```diff
@@ -0,0 +1,290 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace FakerDotNet.Data
+{
+    internal static class CreditCardData
+    {
+
+
+        public static readonly IEnumerable<string> CountryISOCoddes = new[]
```

Comment:
Just a typo on this one.

```suggestion
        public static readonly IEnumerable<string> CountryISOCodes = new[]
```